_plugins/jekyll-notranslate.rb

```diff
@@ -0,0 +1,17 @@
+module Jekyll
+  module NoTranslate
+    def no_translate(text, words)
+        for word in words do
+            word = word.downcase
```

Comment:
You can probably use a regular expression. Something like this:

```ruby
text.gsub(/#{Regexp.escape(word)}/i, "<span class=\"notranslate\">\1</span>")
```

Reply:
ok for the escape and regular expression. Can I combine all the words and downcase/capitalize/upcase changes?

Reply:
Yes.

Reply:
I am failing at implementing that....

Reply:
```
irb(main):007:0> words = ['foo', 'bar']
=> ["foo", "bar"]
irb(main):008:0> escaped_words = words.map { |word| Regexp.escape(word) }
=> ["foo", "bar"]
irb(main):009:0> text = 'hello, foo, this is bar, haha'
=> "hello, foo, this is bar, haha"
irb(main):010:0> text.gsub(/(#{escaped_words.join("|")})/i, "<span class=\"notranslate\">\\1</span>")
=> "hello, <span class=\"notranslate\">foo</span>, this is <span class=\"notranslate\">bar</span>, haha"
irb(main):011:0>
```

(`irb` is like the `python` shell)